splice_machine/src/main/java/com/splicemachine/replication/ReplicationSystemProcedure.java

```diff
@@ -61,6 +62,19 @@
 
     private static Logger LOG = Logger.getLogger(ReplicationSystemProcedure.class);
 
+    public static void GET_REPLICATION_PROGRESS(ResultSet[] resultSets) throws StandardException, SQLException {
+        ReplicationManager replicationManager = EngineDriver.driver().manager().getReplicationManager();
+        long ts = replicationManager.getReplicationProgress();
```

Comment:
It seems that we assume only one replication slave cluster. Can this logic be extended in the future if we have  multiple slave clusters?

Reply:
This system procedure is run on a slave cluster

Reply:
I see. Thanks!